themes/robbyrussell/robbyrussell.theme.sh

```diff
@@ -0,0 +1,25 @@
+#! bash oh-my-bash.module
```

Comment:
Could you mention in a comment that the design is taken from robbyrussell of ohmyzsh?